Tests/YCoreUITests/Extensions/UIKit/UIView+constrainSizeTests.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  UIView+constrainSizeTests.swift
+//  YCoreUITests
+//
+//  Created by Dharmik Ghelani on 17/10/22.
+//  Copyright © 2022 Y Media Labs. All rights reserved.
+//
+
+import XCTest
+@testable import YCoreUI
+
+final class UIViewConstrainSizeTests: XCTestCase {
+    
+    let randomSize = CGSize(width: 150, height: 200)
+    
+    func testSize() {
+        let sut = makeSUT()
+        
+        XCTAssert(sut.translatesAutoresizingMaskIntoConstraints)
+        
+        let sizeAttributes: [
+            (NSLayoutConstraint.Attribute, CGFloat)
+        ] = [
+            (.width, randomSize.width),
+            (.height, randomSize.height)
+        ]
+        
+        let constraints = sut.constrainSize(randomSize)
+        
+        sizeAttributes.forEach {
```

Comment:
How about adding
```
XCTAssertFalse(sut.translatesAutoresizingMaskIntoConstraints)
```

in each of the 3 tests after calling constrain?

Reply:
Correct, done as suggested